src/containing_scope.py

```diff
@@ -0,0 +1,56 @@
+from talon import Context, Module
```

Comment:
we should align on whether we want the file names to be singular or plural, because this one is called `containing_scope` and the other is called `compound_target`.  I think I'd lean toward singular to mirror the name of the capture / list?

Reply:
Also, should we put all the transformations into a `modifiers/` directory?

Reply:
I'll do a PR later to replace every reference to "transformation" to "modifier", but no reason not to use the term for a directory name now

Reply:
I prefer singular. A modifiers directory sounds like a good idea.